jerry-core/lit/lit-magic-strings.h

```diff
@@ -32,6 +32,7 @@ typedef enum
 #undef LIT_MAGIC_STRING_FIRST_STRING_WITH_SIZE
 /** @endcond */
   LIT_NON_INTERNAL_MAGIC_STRING__COUNT, /**< number of non-internal magic strings */
+  LIT_INTERNAL_MAGIC_STRING_ARRAY_PROTOTYPE_VALUES, /**< %ArrayProto_values% intrinsic routine */
```

Comment:
Is it ok to put it here? The next internal magic string will have the previous value.

Reply:
ohh, good catch